Source/SwiftLintFramework/Rules/Lint/EmptyXCTestMethodRule.swift

```diff
@@ -43,16 +57,18 @@ private final class EmptyXCTestMethodRuleVisitor: SyntaxVisitor, ViolationsSynta
             violationPositions.append(node.funcKeyword.positionAfterSkippingLeadingTrivia)
         }
     }
+    
+    private func isNodeATestCase(_ node: ClassDeclSyntax) -> Bool {
```

Comment:
Could be a computed variable of the extension for `ClassDeclSyntax` below replacing `inheritedTypes`.

Reply:
So I ended up with 

```
private extension ClassDeclSyntax {
    var isXCTestCase: (Set<String>) -> Bool {
        { testParentClasses in
            guard let inheritanceList = inheritanceClause?.inheritedTypeCollection else {
                return false
            }
            let inheritedTypes = inheritanceList.compactMap { $0.typeName.as(SimpleTypeIdentifierSyntax.self)?.name.text }
            return testParentClasses.intersection(inheritedTypes).isNotEmpty
        }
    }
}
```

Reply:
I see, the list of test parent classes is needed of course. So a computed property is probably not the right solution to pick. Better to go either with a method in the extension accepting the set or keep the method you already defined. Sorry for missing this.

Reply:
No worries, went with a method